ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteExporter.java

```diff
@@ -685,4 +716,34 @@ private static double getPermanentLimit(Branch branch) {
 
         return Double.min(permanentLimit1, permanentLimit2);
     }
+
+    public static NamingStrategy findNamingStrategy(String name, List<NamingStrategy> namingStrategies) {
+        Objects.requireNonNull(namingStrategies);
+
+        if (namingStrategies.isEmpty()) {
+            throw new PowsyblException("No naming strategy found");
+        }
```

Comment:
Probably never empty because the default implementation is in this module

Reply:
I fixed it and put the method private package, to avoid any incorrect usage that could contain empty list